shared/elements/gaia_grid/style.css

```diff
@@ -374,3 +374,9 @@
 .edit-mode.hover-over-top {
   transform: translateY(0.8rem);
 }
+
+/* RTL View */
+:-moz-dir(rtl) .group .header .toggle {
+  left: 0;
+  right: unset;
```

Comment:
AFAIK only unset behaves this way :)
https://developer.mozilla.org/en-US/docs/Web/CSS/unset
Best of all, applicable to all properties.